README.yaml

```diff
@@ -0,0 +1,62 @@
+---
+#
+# This is the canonical configuration for the `README.md`
+# Run `make readme` to rebuild the `README.md`
+#
+
+# Name of this project
+name: default-backend
+
+# Logo for this project
+#logo: docs/logo.png
+
+# License of this project
+license: "APACHE2"
+
+# Canonical GitHub repo
+github_repo: cloudposse/default-backend
+
+# Badges to display
+badges:
+  - name: "Build Status"
+    image: "https://travis-ci.org/cloudposse/default-backend.svg?branch=master"
+    url: "https://travis-ci.org/cloudposse/default-backend"
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/default-backend.svg"
+    url: "https://travis-ci.org/cloudposse/default-backend/releases"
```

Comment:
Wrong target url. Should be GitHub release page. I haven’t paid attention to this closely in other PRs. Hope not copy and paste mistake.